zerver/webhooks/newrelic/doc.md

```diff
@@ -1,27 +1,70 @@
 
-1. Navigate to **Notification channels**.
+1. Set **Endpoint URL** to the URL you generated.
 
-1. Create a new notification channel. Select channel type of **Webhook**, choose a name (e.g. "Zulip"), enter the webhook url created earlier as **Base Url**.
+1. Go to the [**Workflows**](https://one.newrelic.com/alerts-ai/workflows) menu and click on **+ Add a Workflow** button. After filtering the trigger condition, select on the **Webhook** option in the **Add Channel** section. ([*Workflow doc*](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/))
 
-1. It should look like:
-  ![](/static/images/integrations/newrelic/newrelic.png)
+1. Once you're in the **Edit notification message** menu, select the destination for Zulip you've just created earlier.
 
-1. The webhook works with the default payload, click **Create channel**.
+1. If you want to include additional information to the notification, refer to this [section](#displaying-custom-fields-in-payload). There's also an optional [base payload template](#recommended-base-payload-template) we recommend you to use.
 
-1. After creating the channel send a test notification to make sure it works.
+1. Select **Save message** and the **Activate Workflow** to finish.
+{end_tabs}
 
 {!congrats.md!}
 
 ![](/static/images/integrations/newrelic/001.png)
+
 ![](/static/images/integrations/newrelic/002.png)
-![](/static/images/integrations/newrelic/003.png)
+
+### Displaying Custom Fields
+
+The New Relic integration supports the [custom payload feature](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/message-templates/) offered by New Relic.
+To include your custom fields in your Zulip New Relic notification, follow these rules:
+
+* Put all your custom key-value pairs in a new dictionary called `zulipCustomFields`.
+
+* The Key must be in Camel Case format. Keep in mind that it will be reformatted and
+    displayed in the notification message, so use descriptive words.
+
+* The supported data-type for the custom payload are: `string`, `list`, `int` and `bool`.
+    Custom payload containing `dict` are not supported
+
+    *example custom payload inside `zulipCustomFields`:*
+    ![](/static/images/integrations/newrelic/custom_payload_example.png)
+
+    *the Zulip notification from the payload above:*
+    ![](/static/images/integrations/newrelic/custom_payload_notif.png)
+
+Only custom fields inside the `zulipCustomFields` dictionary will be displayed
+
+### Recommended  Base Payload Template
+
+Although the out of the box payload template is already sufficient
+to get the integration working, the additional fields provided from
+this template will enable the integration to notify you of any
+**acknowledged** incidents:
+
+(THIS IMAGE IS TEMPORARYY, placeholder,)
+![](/static/images/integrations/newrelic/temporary_payload_example.png)
+
+[1]: ./zapier
```

Comment:
Are these links being used in this file?

Reply:
Nope, I was sloppy and forgot to clean them up :sweat_smile: